kedro/templates/project/{{ cookiecutter.repo_name }}/README.md

```diff
@@ -4,14 +4,14 @@
 
 This is your new Kedro project, which was generated using `Kedro {{ cookiecutter.kedro_version }}`.
 
-Take a look at the [Kedro documentation](https://kedro.readthedocs.io) to get started.
+Take a look at the [Kedro documentation](https://docs.kedro.org) to get started.
 
 ## Rules and guidelines
 
 In order to get the best out of the template:
 
 * Don't remove any lines from the `.gitignore` file we provide
-* Make sure your results can be reproduced by following a [data engineering convention](https://kedro.readthedocs.io/en/stable/faq/faq.html#what-is-data-engineering-convention)
+* Make sure your results can be reproduced by following a data engineering convention
```

Comment:
🕵🏽 Do we have an issue tracking how do we want to treat this "convention" more broadly in our docs?

Reply:
I guess we could discuss that in https://github.com/kedro-org/kedro/issues/2379

Reply:
As you know, I've removed it from our docs except to point to the canonical blog post, mostly to deal with the removal of that Q&A pair from the FAQs. One of the upcoming posts we'll put on the Kedro blog will deal with data layers and, when that is published, I plan to replace the Medium blog links with the newer post. 

I don't know, I see this as more of a philosophy/opnion that docs should not be espousing, but can direct attention towards. WDYT @astrojuanlu ?

Reply:
> I don't know, I see this as more of a philosophy/opnion that docs should not be espousing, but can direct attention towards.

I 💯 % agree with this.